ShopperInsights/src/main/java/com/braintreepayments/api/shopperinsights/v2/ShopperInsightsClientV2.kt

```diff
@@ -123,4 +125,26 @@ class ShopperInsightsClientV2 internal constructor(
     fun isVenmoAppInstalled(context: Context): Boolean {
         return deviceInspector.isVenmoInstalled(context)
     }
+    /**
+     * Updates the customer session with the given session ID.
+     */
+    fun updateCustomerSession(
+        customerSessionRequest: CustomerSessionRequest,
+        sessionId: String,
+        callback: (UpdateCustomerSessionResult) -> Unit
+    ) {
+        updateCustomerSessionApi.execute(customerSessionRequest, sessionId) { result ->
+            when (result) {
+                is UpdateCustomerSessionApi.UpdateCustomerSessionResult.Success ->
+                    callback(UpdateCustomerSessionResult.Success(result.sessionId))
+                is UpdateCustomerSessionApi.UpdateCustomerSessionResult.Error ->
+                    callback(UpdateCustomerSessionResult.Failure(result.error))
+            }
+        }
+    }
+
+    sealed class UpdateCustomerSessionResult {
```

Comment:
I think this sealed class matches the existing [CustomerSessionResult](https://github.com/braintree/braintree_android/blob/payment-ready-v2/ShopperInsights/src/main/java/com/braintreepayments/api/shopperinsights/v2/CustomerSessionResult.kt) class. Should we reuse this class? (if we do, we may need to update some of the kdocs as it calls out that it's for creating a session)